module/item/item-attack.mjs

```diff
@@ -1356,6 +1357,7 @@ export async function _onRollDamage(event) {
         .makeAdjustmentRoll(!!adjustment)
         .makeFlashRoll(!!senseAffecting)
         .makeEntangleRoll(entangle)
+        .makeTransformRoll(transform)
```

Comment:
My assumption was that this would go away and this would be something like `makeEffectRoll(transform)`. Again this can wait until after this PR.